### PR TITLE
run acc for go file change only

### PR DESCRIPTION
**Pull request by byashimov**
Run acc for go file change only

### CODE REVIEW

File: .github/workflows/acceptance-tests-ignore.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  push:
+    paths-ignore:
+      - '**.go'
+  workflow_dispatch:
+
+jobs:
+  acceptance_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - run: 'echo "No build required"'
```
Comment (ivan-savciuc) [inline .github/workflows/acceptance-tests-ignore.yaml:14]: Please end all files with an empty line

**Review by ivan-savciuc — CHANGES_REQUESTED** [review]
LGTM, we just need to add a new empty line I guess

File: .github/workflows/acceptance-tests-ignore.yaml (hunk 1 above)
Comment (byashimov) [reply .github/workflows/acceptance-tests-ignore.yaml]: Done

**Review by ivan-savciuc — APPROVED** [review]
LGTM

**Pull request merged**